Resources/config.xml

```diff
@@ -42,5 +42,24 @@
             <label>Mountly limit</label>
             <value>500</value>
         </element>
+        <element type="select">
+            <name>mimeTypes</name>
+            <label>Use for image formats</label>
+            <label lang="de">Nutze für Bildformate</label>
+            <value>image/png</value>
+            <store>
+                <option>
+                    <value>image/png</value>
+                    <label>PNG</label>
+                </option>
+                <option>
+                    <value>image/jpeg</value>
```

Comment:
Thanks for your work! This should result for savety in image/jpg, too.

Reply:
Nope. Won't fix. Haha. Ever wanted to say that.
There is no mime type image/jpg. Basicly, PHP is using parts of the `file` command to guess the mime type which is based in the following src files:
https://git.in-ulm.de/cbiedl/file/src/master/magic/Magdir/jpeg#L13
For uploading file in rare cases the Symfony HTTP Foundation component is used, which is based on an apache source:
http://svn.apache.org/repos/asf/httpd/httpd/trunk/docs/conf/mime.types

Reply:
Passed all tests :trollface: 
Thank you!